### PR TITLE
Update Tickets class to use datetime as input

### PR DESCRIPTION
**Pull request by qubixes**
I have modified the Tickets class a little bit to make it slightly easier to work with (in my opinion). Feedback is of course welcome.

One thing that I ran into is that although a datetime is used as input for the tickets, it returns a timestamp at midnight. Is that expected behavior?

### CODE REVIEW

File: Tutorial_irodsConnector.ipynb [hunk 1]
```diff
@@ -720,9 +720,8 @@
    "source": [
     "from datetime import datetime\n",
     "\n",
-    "exp_date = datetime.today().strftime('%Y-%m-%d.%H:%M:%S')\n",
     "objPath = \"/nluu12p/home/research-test-christine/books/BenHur.txt\"\n",
-    "ticket = tickets.create_ticket(obj_path=objPath, ticket_type=\"write\", expiry_string=exp_date) # allow write access"
+    "ticket = tickets.create_ticket(obj_path=objPath, ticket_type=\"write\", expiry_date=datetime.today())  # allow write access"
```
Comment (chStaiger) [inline Tutorial_irodsConnector.ipynb:724]: `"ticket = tickets.create_ticket(obj_path=objPath, ticket_type=\"write\", expiry_date=datetime.date.today() + datetime.timedelta(days=1))  # allow write access"`

Comment (chStaiger) [inline Tutorial_irodsConnector.ipynb:724]:
```suggestion
    ""ticket = tickets.create_ticket(obj_path=objPath, ticket_type=\"write\", expiry_date=datetime.date.today() + datetime.timedelta(days=1))  # allow write access""
```

**Review by chStaiger — CHANGES_REQUESTED** [review]
Many thanks, that is indeed much more user-friendly ;). I left a suggestion for even more user-friendlyness. 
I also saw that the unit test on Yoda is failing. Is it possible to get that to work too?

**Comment by qubixes** [comment]
> Many thanks, that is indeed much more user-friendly ;). I left a suggestion for even more user-friendlyness. I also saw that the unit test on Yoda is failing. Is it possible to get that to work too?

That is basically my question to you. It seems that all tickets are expiring at midnight. Is that the expected/correct behavior? It seems counter-intuitive.

File: Tutorial_irodsConnector.ipynb (hunk 1 above)
[the remark this answers is not in the document]
Comment (qubixes) [reply Tutorial_irodsConnector.ipynb]: Yes, that makes more sense!

**Pull request merged**